vminitd/Sources/vminitd/ManagedProcess.swift

```diff
@@ -143,131 +149,151 @@ final class ManagedProcess: Sendable {
         self.command = command
         self.terminal = stdio.terminal
         self.bundle = bundle
-        self.state = Mutex(State(io: io))
+        self.state = Mutex(State(io: io, errorPipe: errorPipe))
```

Comment:
Sorry, I kind of inadvertently recommended this, but I'd forgotten how we'd done the other two pipes. Can you just instantiate the error pipe in the constructor, and then just have it be a member variable of this type just like syncPipe and ackPipe. I don't think we need to lock around it.